index.html

```diff
@@ -41,5 +41,181 @@ <h1>I’m Marcelo
         </ul>
     </div>
 </section>
+<!---->
+<section class="work">
+    <!-- project-1 -->
+    <div class="project-1 projects">
+        <div class="card">
+            <span><img src="/imagenes/project_1.png" alt="project_1"></span>
+            <span><h2>Tonic</h2></span>
```

Comment:
[Optional] Good job putting the project name into a heading tag, but I would like to recommend removing the `span` tag and keeping it clean.